go.mod

```diff
@@ -25,22 +25,22 @@ require (
 	github.com/lightninglabs/lightning-terminal/autopilotserverrpc v0.0.3
 	github.com/lightninglabs/lightning-terminal/litrpc v1.0.2
 	github.com/lightninglabs/lightning-terminal/perms v1.0.1
-	github.com/lightninglabs/lndclient v0.19.0-12
-	github.com/lightninglabs/loop v0.31.2-beta
-	github.com/lightninglabs/loop/looprpc v1.0.8
-	github.com/lightninglabs/loop/swapserverrpc v1.0.15
+	github.com/lightninglabs/lndclient v0.20.0-1
+	github.com/lightninglabs/loop v0.31.3-beta.0.20251014011913-03de0a8ed734
+	github.com/lightninglabs/loop/looprpc v1.0.10-0.20251009095951-719355211f63
+	github.com/lightninglabs/loop/swapserverrpc v1.0.17-0.20251009095951-719355211f63
 	github.com/lightninglabs/pool v0.6.6-beta
 	github.com/lightninglabs/pool/auctioneerrpc v1.1.3
 	github.com/lightninglabs/pool/poolrpc v1.0.1
-	github.com/lightninglabs/taproot-assets v0.6.1
-	github.com/lightninglabs/taproot-assets/taprpc v1.0.8-0.20250716163904-2ef55ba74036
-	github.com/lightningnetwork/lnd v0.19.3-beta
+	github.com/lightninglabs/taproot-assets v0.7.0-rc1.0.20251014100421-92189161aa7a
```

Comment:
Also, with this, we'll need to wait for `tapd` `v0.7.0-rc2` before we can do a release as well.